native/cocos/renderer/pipeline/custom/details/PathUtils.h

```diff
@@ -42,7 +41,7 @@ inline void cleanPath(std::basic_string<CharT, std::char_traits<CharT>, Allocato
     constexpr CharT doubleSlash[] = {'/', '/', '\0'};
 
     CC_EXPECTS(!str.empty());
-    CC_EXPECTS(boost::algorithm::starts_with(str, std::string_view(slash)));
+    CC_EXPECTS(std::string_view{str}.substr(0, 1) == slash);
```

Comment:
Note that, std::string_view is not well supported on iOS 12.

Reply:
Maybe I forgot that. It's std::optional was not supported on iOS 12, so we use `ccstd::optional` which wraps `boost::optional`.
I found there are lots of `std::string_view` usage in our engine, so it should be safe to use it.

Reply:
Yes, `std::string_view` is supported. `optional` is not much used, but I will take care.